.github/workflows/issue-inactive.yml

```diff
@@ -2,7 +2,8 @@ name: Issue Inactive
 
 on:
   schedule:
-    - cron: '0 8 * * *'
+    # GMT+8 00:00
+    - cron: '0 16 * * *'
```

Comment:
I think the express is wrong for your intention.

check out this website: https://crontab.guru/

For your expression means:  it occurs every day at 16:00 of the day, since GMT + 8 is 8 hours earlier than GMT + 0 (Greenwich time)